### ISSUE TITLE
Unable to clone : cannot create directory at '"if" statements': Invalid argument

### ISSUE THREAD

**Issue by calebhillary97** [pirple/Python-Is-Easy#1]
I can't clone this repo. I am getting this error
cannot create directory at '"if" statements': Invalid argument

I use windows and found out that the quotes(") found in the folder name '"if" statements' is not a valid one in windows. Windows doesn't allow some of these special characters in file/folder names.

Consider renaming them to better suit a wide range of students.

**Comment by Bankole2000** [comment]
Yeah... had this same issue too... didn't think it was a big deal so I just downloaded and extracted the zip file... would've been nice to have it cloned though... keep it updated n such...
